### PR TITLE
Update OpenCost readme

### PR DESCRIPTION
**Pull request by dwbrown2**
## What does this PR change?
* Point to new install steps + FAQ page.

## Does this PR relate to any other PRs?
* No 

## How will this PR impact users?
* Just change links + copy in README

## Does this PR address any GitHub or Zendesk issues?
* None

## How was this PR tested?
* Previewed changes and followed links

## Does this PR require changes to documentation?
* Yes, done!

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Kubecost release? If not, why not?
*

### CODE REVIEW

**Comment by dwbrown2** [comment]
@AjayTripathy @Adam-Stack-PM any chance you can help me get a review here?

**Review by Adam-Stack-PM — APPROVED** [review]
This looks good to me.  FAQ content was reviewed at a high level. Ping me if you want me to track down a deep dive for you.

**Comment by dwbrown2** [comment]
Thanks, @Adam-Stack-PM! Still need maintainer approval here.

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**
